src/components/SideMenu/SideMenu.js

```diff
@@ -40,13 +40,13 @@ export function SideMenu ({
             </Link>
           </li>
           <li>
-            <Link to='/browse-happs' styleName='nav-link'>
+            <Link to='/browse-happs' styleName='nav-link' data-testid='hosting-link'>
```

Comment:
we want to avoid adding `data-testid` wherever possible. Can't both of these links be found using

`getByText('Hosting')` and `getByText('Earnings')`

?

Reply:
I did that originally, but the for 'Hosting' and ''Earnings" link names are also mirrored in the header sub-title and thereby cause an error when searching for a single single listing.  We could use the 'getAll' matchers here instead and just anticipate thesub-title or sidenav link names index order when referencing them in tests - ie: `getAllByText'('Hosting')[0]`, but that route can be less deterministic and a bit less intuitive.  Thoughts?